Mathlib/Analysis/InnerProductSpace/Orthogonal.lean

```diff
@@ -138,7 +138,7 @@ theorem isClosed_orthogonal : IsClosed (Kᗮ : Set E) := by
 #align submodule.is_closed_orthogonal Submodule.isClosed_orthogonal
 
 /-- In a complete space, the orthogonal complement of any submodule `K` is complete. -/
-instance instOrthogonalCompleteSpace [CompleteSpace E] : CompleteSpace Kᗮ :=
+instance Orthogonal.completeSpace [CompleteSpace E] : CompleteSpace Kᗮ :=
```

Comment:
We're currently (but far from consistently) aiming to have `inst` at the beginning of all manually named instances.